test/test_upnp.cpp

```diff
@@ -46,17 +46,25 @@ using namespace libtorrent;
 broadcast_socket* sock = 0;
 int g_port = 0;
 
-char soap_add_response[] =
+char soap_add_response[][270] = {
```

Comment:
I believe you could also say: char const* soap_add_request[] = 
to not have to specify the length of the strings